tests/test_jobs.py

```diff
@@ -201,9 +201,13 @@ def test_start_four_services_in_order(self, mock_docker_run):
 
         mock_docker_run.assert_has_calls([
             mock.call('sudo service redis-server start'),
+            mock.call().succeeded.__bool__(),
```

Comment:
Since the mock returns a MagickMock all calls to any attribute of that object will be in the call list, thus if statement in [`if not self.docker.run('sudo service {0} start'.format(service)).succeeded:`](https://github.com/frigg/frigg-worker/pull/50/files#diff-259a6cde6c4c0cbdb0dd42c579112a1eR149) will result in `mock.call().succeeded.__bool__()`